mmv1/products/compute/Network.yaml

```diff
@@ -124,7 +124,7 @@ properties:
     validation:
       function: 'verify.ValidateGCEName'
   - name: 'numericId'
-    type: String
+    type: Integer
```

Comment:
There are some corner cases to do with modules, unfortunately, that mean these corrections are breaking :(

It's probably best to add this as a string in the datasource as well for consistency, and we can track flipping the types in the next major release.

Reply:
Ok, I can update it. Just weird that network and subnetwork won’t be consistent with each other.

Might be good to keep track somewhere of all the fields that are string but should be int, in the board you have setup for breaking changes?